### DISCUSSION TITLE
打包前做版本判断

### DISCUSSION THREAD

**Discussion by simplast** [umijs/umi#10619] [Q&A] [answered]
1，打包完成后在dist里面写入一个svn的版本号
2，下次打包前获取当前svn版本和dist里面的版本，如果一致就跳过打包

问题是怎么跳过打包？

**Comment by fz6m** [comment, marked as answer]
写一个构建脚本，在这个脚本里运行 umi build ，去读文件识别就行。

**Reply by simplast** [reply]
好像是这么回事。。

**Comment by xiaohuoni** [comment]
插件的话，用 api. onCheck 判断之后中断命令也可以
